nixos/modules/misc/ids.nix

```diff
@@ -295,6 +295,7 @@
       aria2 = 277;
       clickhouse = 278;
       rslsync = 279;
+      piwik = 280;
```

Comment:
My initial thought was so that the whole `dataDir` folder `/var/lib/piwik` can be easily restored from a backup, e.g. after a failure that needed a reinstall or on another machine.

In the meantime I learned that actually only a single file, `/var/lib/piwik/config/config.ini.php` needs to be backed up. It's probably easy enough to fix uid+gid for a single file manually on restores, so I guess it's not *that* required anymore. Nothing in the code depends on it, I could simply remove the static assignment again.

This is my first nixpkg package, could you tell me more about in which cases and for what purposes it's considered good pratice to use static ids, and in which it isn't?